package.json

```diff
@@ -53,6 +53,7 @@
     "ts-jest": "24.3.0",
     "typescript": "4.2.3",
     "typescript-json-schema": "0.42.0",
+    "uint8array-json-parser": "^0.0.1",
```

Comment:
Why would adding a single dependency add 15000 lines to the package-lock.json file?

Reply:
Huh. It's a good question.

My guess is something to do with the `lockfileVersion` change: https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json#lockfileversion

Before this change, the `package-lock.json` was `lockfileVersion: 1`, and after the chang eit was `lockfileVersion: 2`. This has nothing to do with this change, it just so happens this is the first time I've installed something since the npm version change.